opencog/reasoning/pln/rules/inheritance-to-member.scm

```diff
@@ -0,0 +1,51 @@
+; =============================================================================
+; InheritanceToMemberRule
+;	InheritanceLink( B C )
+;			becomes
+;	MemberLink( B C )
+; -----------------------------------------------------------------------------
+
```

Comment:
@ngeiswei : Should I incorporate setlinks on all the rules you mentioned? Also, I would also suggest to reflect the necessary changes in the [socrates examples readme](https://github.com/opencog/opencog/tree/master/opencog/python/pln_old/examples/socrates_demo). Can I do that as well?